FluidNC/src/Report.cpp

```diff
@@ -433,7 +433,7 @@ void report_build_info(const char* line, Print& channel) {
     channel << "[VER:" << grbl_version << " FluidNC " << git_info << ":" << line << "]\n";
     channel << "[OPT:";
     if (config->_coolant->hasMist()) {
-        channel << "M";  // TODO Need to deal with M8...it could be disabled
```

Comment:
The status of the cooling doesn't matter since OPT: reports compile time (or in this case runtime config) options and not the actual status.